src/20_Components/amp-accordion.html

```diff
@@ -74,5 +74,44 @@
         <p>Id lacus amet. Aliquam eos nunc ut scelerisque lacinia, eu rutrum id, vestibulum aliqua vivamus luctus eu rhoncus ut, sodales id. Velit lacus, fermentum neque et sagittis, ac venenatis volutpat, dolore neque feugiat proin fermentum odio, odio arcu in eu wisi. </p>
       </section>
     </amp-accordion>
+
+    <!-- #### Nested Accordions -->
+    <!-- 
+    You can even nest multiple accordions. Just be sure to avoid styling elements based on the expanded state of the enclosing amp-accordion.  In particular, avoid overly generic rules such as:
+
+    ```css
+    section[expanded] h4 {
+      ...
+    }
+    ```
+
+    instead directly target elements:
+  
+    ```css
+    section[expanded] > h4 {
+      ...
+    }
+    ```
+    
+    -->
+    <amp-accordion >
+      <section>
+        <h4>Section 1</h4>
+        <p>Bunch of content.</p>
+      </section>
+      <section>
+        <h4>Section 2</h4>
+        <amp-accordion>
```

Comment:
would it be possible to have a different style (such as changing the font or the color) for the nested content? Just to be visually clearer